File: Even.cpp
#include<iostream>
using namespace std;
class Number
{
    int num;
    public: void check()
    {
        cout<<"Number is \n";
        cin>>num;
        if(num%2==0)
        {
            cout<<"Number is Even \n";
        }
        else{
            cout<<"Numbers is odd \n";
        }
    }
};
int main()
{
    Number n1;
    n1.check();
}

File: JAVA/All Codes/Even.java
import java.util.Scanner;
class Even
{
public static void main(String args[])
{
int num;
System.out.println(".........................................................");
System.out.println("Enter the Number:");
Scanner s=new Scanner(System.in);
num=s.nextInt();
if(num%2==0)
{
System.out.println(".........................................................");
System.out.println("Entered Number is Even"+num);
}
else
{
System.out.println(".........................................................");
System.out.println("Entered Number is Odd");
}
}
}